tests/tally/generator.rs

```diff
@@ -168,8 +178,8 @@ impl VoteRoundGenerator {
         };
     }
 
-    fn token_distribution(&self, ledger: &Ledger) -> TokenDistribution<()> {
-        let token_totals = TokenTotals::default()
+    fn token_totals(&self, ledger: &'a Ledger) -> TokenTotals {
```

Comment:
is this actually necessary? you can call `token_distribution` directly on the ledger for the things this is used